### DISCUSSION TITLE
New pattern for Ushanka style hat

### DISCUSSION THREAD

**Discussion by darigovresearch** [freesewing/freesewing#1619] [Forum]
**What is it that you would like to see happen?**

Ushanka hats are simple and versatile soft hat for cold weather climates and may be a simple first hat for people who are looking to get started which can be adapted in various ways. For example wool fleece can be used as when wet, retains 80% of it's insulative properties.

https://en.wikipedia.org/wiki/Ushanka

**Are you a FreeSewing patron?**

 - [ ] Yes, I am :hugs:
 - [x] No, I am not :thinking:

**Additional context**

This will likely be one of our first pattern to see what the contribution process is like & how to generate parametric patterns. Anybody who is new to contributing or interested in helping is welcome to join. See this video if you need some guidance on how you may be able to help - https://www.youtube.com/watch?v=UWA4wyacY2A

Suggested name is Ulysses the Ushanka hat

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions

**Comment by darigovresearch** [comment]
Reopening discussion from the community call https://github.com/freesewing/freesewing/discussions/5532

**Comment by BenJamesBen** [comment]
Closing because there does not seem to be any interest for active discussion about this topic, per the April 27, 2024 Community Call. https://github.com/freesewing/freesewing/discussions/5724#discussioncomment-9247197

(Please feel welcome to re-open this Discussion if there is interest in resuming active discussion about the topic.)
